src/CacheManager.Web/SystemWebCacheHandle`1.cs

```diff
@@ -86,6 +89,20 @@ public override bool Exists(string key, string region)
             return GetCacheItemInternal(key, region) != null;
         }
 
+        /// <inheritdoc />
+        protected override IEnumerable<string> AllKeys()
+        {
+            var keys = new List<string>();
+
+            var item = Context.Cache.GetEnumerator();
+            while (item.MoveNext())
+            {
+                keys.Add(item.Key.ToString());
```

Comment:
yield wouldn't gain anything here as the caller iterates over the list when doing the pattern match.

If System.Web.Cache has significantly different performance than the other in-memory caches, maybe it should not implement this option.